src/test/TestUtils/TestProjectFixture.cs

```diff
@@ -277,16 +277,23 @@ private void ValidateRequiredDirectories(RepoDirectoriesProvider repoDirectories
             return this;
         }
 
-        public TestProjectFixture RestoreProject(string[] fallbackSources)
+        public TestProjectFixture RestoreProject(string[] fallbackSources, string extraMSBuildProperties = null)
         {
             var restoreArgs = new List<string>();
             foreach (var fallbackSource in fallbackSources)
             {
-                restoreArgs.Add("-f");
+                restoreArgs.Add("--source");
                 restoreArgs.Add(fallbackSource);
             }
             restoreArgs.Add("--disable-parallel");
 
+            restoreArgs.Add($"/p:MNAVersion={_repoDirectoriesProvider.MicrosoftNETCoreAppVersion}");
```

Comment:
How about instead of making up a new property for this, instead just set the property that is already used for it - `$(RuntimeFrameworkVersion)`?